test/csuite/timestamp_abort/main.c

```diff
@@ -413,10 +424,9 @@ thread_run(void *arg)
         cur_local->set_value(cur_local, &data);
         testutil_check(cur_local->insert(cur_local));
 
-        /*
-         * Save the timestamp and key separately for checking later.
-         */
-        if (fprintf(fp, "%" PRIu64 " %" PRIu64 "\n", active_ts, i) < 0)
+        /* Save the timestamps and key separately for checking later. */
+        if (fprintf(fp, "%" PRIu64 " %" PRIu64 " %" PRIu64 "\n", active_ts,
+              durable_ahead_commit ? active_ts + 4 : active_ts, i) < 0)
             testutil_die(EIO, "fprintf");
```

Comment:
@sueloverso The previous attempt has an issue. I now record both the commit timestamp and durable timestamp in the external log file.

Reply:
The recovery stress test now is green with the fix.